.gitignore

```diff
@@ -124,4 +124,6 @@ venv.bak/
 napari/resources/_qt_resources*.py
 napari/resources/res.qrc
 # ignore all generated themed svgs
-napari/resources/themes
\ No newline at end of file
+napari/resources/themes
+
+napari/_version.py
```

Comment:
Interesting...? 😬

Reply:
yeah, we don't want to check that in.  basically, `napari/_version.py` will be written by `setuptools_scm` when you run `pip install -e .` or when you run `python setup.py sdist bdist_wheel`, and will create `napari/_version.py` at that moment.  (See also, the comment below)

here's an example of how it looks:
```python
# coding: utf-8
# file generated by setuptools_scm
# don't change, don't track in version control
version = '0.3.1.dev55+g676d8584.d20200602'
```